pkgs/os-specific/linux/bluez/default.nix

```diff
@@ -1,13 +1,13 @@
 { stdenv, fetchurl, pkgconfig, dbus, glib, alsaLib,
-  python3, readline, udev, libical, systemd, fetchpatch,
+  python3, readline, udev, libical, systemd,
   enableWiimote ? false, enableMidi ? false, enableSixaxis ? false }:
 
 stdenv.mkDerivation rec {
-  name = "bluez-5.50";
+  name = "bluez-5.51";
```

Comment:
Is it possible to switch to `pname` and `version`? I think it is useful to have a separate version field.

Doing that while we update to a new version might prevent rebuilding a lot of packages for a trivial change.